Dockerfile

```diff
@@ -0,0 +1,29 @@
+# Build Stage
```

Comment:
Please don't use git clone or `ldd`. Is `AFL_NO_FORKSRV` necessary here?

Reply:
I had to use AFL_NO_FORKSRV as every run would crash due to an inability to spool up the fork server.

Reply:
Nevermind, did something change with the fork server problem we originally discussed? Re-enabled that and it is blazing fast now